x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/es_index.tsx

```diff
@@ -224,8 +225,7 @@ const IndexActionConnectorFields: React.FunctionComponent<ActionConnectorFieldsP
               content={i18n.translate(
                 'xpack.triggersActionsUI.components.builtinActionTypes.indexAction.definedateFieldTooltip',
                 {
-                  defaultMessage:
-                    'This is checkbox allows to define execution time field for index.',
+                  defaultMessage: `Should a time field be added to each document automatically when it's indexed?`,
```

Comment:
Not sure about '?' at the end, but other changes looks great.

Reply:
Yup, mistake :/

Good catch :)